Importing GeoTIFF source causes compilation errors in Angular

**Describe the bug**
I am using OpenLayers 6.15.1 in an Angular 14 application and when I import the source GeoTIFF I get the following compilation errors:

```
Error: node_modules/geotiff/dist-node/geotiffimage.d.ts:67:118 - error TS2304: Cannot find name 'Source'.
67     constructor(fileDirectory: any, geoKeys: any, dataView: DataView, littleEndian: boolean, cache: boolean, source: Source);

Error: node_modules/geotiff/dist-node/geotiffimage.d.ts:75:13 - error TS2304: Cannot find name 'Source'.
75     source: Source;

Error: node_modules/geotiff/dist-node/geotiffimage.d.ts:134:100 - error TS2304: Cannot find name 'AbstractDecoder'.
134     getTileOrStrip(x: number, y: number, sample: number, poolOrDecoder: import("./geotiff").Pool | AbstractDecoder, signal?: AbortSignal | undefined): Promise<ArrayBuffer>;
```



**To Reproduce**
1. `ng new test-app`
2. `cd test-app`
3. `npm install ol`
4. Add `import GeoTIFFSource from 'ol/source/GeoTIFF';` to app.component.ts
5. `ng build`

**Expected behavior**
No compilation errors.
Please confirm if this should be reported on the _geotiff_ repository instead.

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
   },
   "dependencies": {
     "earcut": "^2.2.3",
-    "geotiff": "2.0.4",
+    "geotiff": "^2.0.7",
     "ol-mapbox-style": "^9.2.0",
     "pbf": "3.2.1",
     "rbush": "^3.0.1"

